### PR TITLE
docs: add postmortem of tx receipts validation skip fork

### PR DESCRIPTION
**Pull request by 0xHansLee**
## Summary by CodeRabbit

- **Documentation**
	- Updated the incident summary header text casing for better readability.
	- Added detailed documentation for the chain halt incident on 2024-04-01, covering causes, impact, recovery steps, and future improvement plans.

### CODE REVIEW

**Review by sm-stack — APPROVED** [review]

**Review by Pangssu — APPROVED** [review]

File: postmortems/2024-04-01-tx-receipts-validation-skip-incident.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# 2024-04-03 Chain Halt due to tx receipts validation skip
```
Comment (seolaoh) [inline postmortems/2024-04-01-tx-receipts-validation-skip-incident.md:1]:
```suggestion
# 2024-04-01 Chain Halt due to Tx Receipts Validation Skip Post-Mortem
```

File: postmortems/2024-04-01-tx-receipts-validation-skip-incident.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Causes
+
+Block head of our L1 RPC began to fall behind the canonical chain, triggering a switch to the fallback L1 RPC once it 
+deviated beyond a certain threshold. During the switch, sequencer attempted to retrieve transaction receipts from the L1
+RPC node that fell behind it had not yet synced, resulting in empty transaction receipt responses.
+
+Actually, the block in question did contain [deposited transaction](https://kromascan.com/tx/0xf76e4f34645bc3e172909fb03311cd1770f9c574ce46bd1da3644f9cea82c0e3)
+on L1, yet the sequencer proceeded to generate the block without processing this transaction, having received empty 
+transaction receipts from the trusted L1 RPC. As a consequence, starting from block 9029744, a different block was 
+generated, leading to a generation of wrong block. This is the transaction not included in the block sequencer
```
Comment (seolaoh) [inline postmortems/2024-04-01-tx-receipts-validation-skip-incident.md:28]: I think a link of the omitted transaction to `This` is missing.

File: postmortems/2024-04-01-tx-receipts-validation-skip-incident.md (hunk 1 above)
Comment (0xHansLee) [reply postmortems/2024-04-01-tx-receipts-validation-skip-incident.md]: Thanks, I applied in [47009b1](https://github.com/kroma-network/kroma/pull/289/commits/47009b1023eb7a05e741326b5c955428aa1ebe88)

File: postmortems/2024-04-01-tx-receipts-validation-skip-incident.md (hunk 2 above)
Comment (0xHansLee) [reply postmortems/2024-04-01-tx-receipts-validation-skip-incident.md]: Added in [47009b1](https://github.com/kroma-network/kroma/pull/289/commits/47009b1023eb7a05e741326b5c955428aa1ebe88)

**Review by seolaoh — APPROVED** [review]
LGTM

**Pull request merged**